core/src/main/scala/chisel3/Num.scala

```diff
@@ -205,9 +206,9 @@ trait NumObject {
     * @param binaryPoint a binaryPoint that you would like to use
     * @return
     */
-  def toBigInt(x: BigDecimal, binaryPoint: Int): BigInt = {
+  def toBigInt(x: BigDecimal, binaryPoint: Int, roundingMode: RoundingMode = HALF_UP): BigInt = {
```

Comment:
Since this is a bugfix, I'd like to backport it. Stable branches check binary compatibility and obnoxiously in Scala (2 at least, not sure about 3) it is not binary compatible to add an argument with a default parameter--instead you have to just overload the method, ie.:
```scala
def toBigInt(x: BigDecimal): BigInt = toBigInt(x, HALF_UP)
def toBigInt(x: BigDecimal, binaryPoint: Int, roundingMode: RoundingMode): BigInt = ...
```
I will push this change to your branch, I just wanted to note it before we merge.

Reply:
I just realized the version without the default argument already exists, I'm surprised this isn't causing a problem...